extensions/move-desktop-screenshots-to-trash/src/index.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const { exec } = require('child_process');
```

Comment:
```suggestion
import fs from "fs";
import os from "os";
import path from "path";
import { exec } from "child_process";
```

Reply:
Tested locally and this works, thank you!